Dockerfile

```diff
@@ -1,11 +1,24 @@
 FROM ubuntu:14.04
 MAINTAINER EverydayHero <edh-dev@everydayhero.com.au>
 
-RUN apt-get update && apt-get install -y  \
-        geoip-database \
-        libgeoip1 \
-        nginx-full \
-    && rm -rf /var/lib/apt/lists/*
+WORKDIR /
+
+ADD https://gist.githubusercontent.com/coop/922941b03a0c2af631a7/raw/935b474edfc77636156a8b4f3c7c7b4d31ad40c1/rules.patch /
+
+RUN apt-get update && \
+    apt-get install -y  \
+    geoip-database \
+    libgeoip1 \
+    wget && \
+    apt-get build-dep nginx-full -y && \
+    apt-get source nginx-full
+
+RUN $(cd nginx-1.4.6/debian/modules && wget https://github.com/streadway/ngx_txid/archive/master.tar.gz -O - | tar -xz) && \
```

Comment:
Should we be hard coding the version in here? wouldn't it make more send to specify the version dynamically (though I see the rules patch has it as well.